codespell_lib/data/dictionary.txt

```diff
@@ -11920,11 +11968,13 @@ reaktivated->reactivated
 realeased->released
 realitime->realtime
 realitvely->relatively
+reall->real
 reallize->realize
 realsitic->realistic
 realted->related
 realtion->relation
 realtions->relations
+realtive->relative
```

Comment:
Suggest reactive

Reply:
In fact the same could apply for most of the realt... ones above!

Reply:
Done and added